src/main/java/com/example/persimmoncocktails/services/StockService.java

```diff
@@ -0,0 +1,73 @@
+package com.example.persimmoncocktails.services;
+
+import com.example.persimmoncocktails.dao.StockDao;
+import com.example.persimmoncocktails.dtos.stock.*;
+import com.example.persimmoncocktails.exceptions.IncorrectNameFormat;
+import com.example.persimmoncocktails.exceptions.IncorrectRangeNumberFormat;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+public class StockService {
+    StockDao stockDao;
+
+    public void delete(Long ingredientId, Long personId) {
```

Comment:
You need to add checks for all methods. For example, checking for the existence of a user or an ingredient.